treetracker-core/Services/User/LoginService.swift

```diff
@@ -33,7 +33,7 @@ class LocalLoginService: LoginService {
 
         let managedObjectContext = coreDataManager.viewContext
         let fetchRequest: NSFetchRequest<PlanterDetail> = PlanterDetail.fetchRequest()
-        fetchRequest.predicate = NSPredicate(format: "identifier == %@", username.value)
+        fetchRequest.predicate = NSPredicate(format: "identifier =[cd] %@", username.value)
```

Comment:
Maybe just add a comment around the use of `=[cd]` here as it won't be obvious to a lot of people

Reply:
@AlexCornforth I didn't convert `username.value` to lowercase version here because the `[cd]` in the NSPredicate  already does that.